.travis.yml

```diff
@@ -28,9 +29,13 @@ matrix:
       dart: dev
     - env: TASK="./tool/check-code.sh"
       dart: dev
+    - env: TASK="bundle exec jekyll build;./tool/shared/check-links.sh"
```

Comment:
Should this be `check-links.sh -e`?

Reply:
That's for checking external paths. What I'm doing here is ensuring that inside this matrix job the site is actually generated first. But I got the syntax wrong.